### PR TITLE
#163341312 admin should be able to delete party

### PR DESCRIPTION
**Pull request by Bobjayafam**
#### What does this PR do?
- Implement a modal that allows admin to delete a party
#### Description of Task to be completed?
- create a modal for delete confirmation
- add javascript functions to open and close modal
#### How should this be manually tested?
- git clone the project repo
- cd UI
- open ``admin.html`` with a browser of your choice, click the delete button beside any political party
#### Any background context you want to provide?
A party no longer exists and needs to be removed from the application
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163341312
#### Screenshots (if appropriate)
delete-party-spacing

### CODE REVIEW

File: UI/css/styles.css [hunk 1]
```diff
@@ -244,21 +244,23 @@ thead tr {
...
+.modal button {
   background: #FF9902;
   padding: 10px;
   display: inline-block;
   border: 0;
   cursor: pointer;
   border-radius: 5px;
   transition: 0.3s;
-  margin-top: 20px;
+  /* margin-top: 20px; */
```
Comment (megame24) [inline UI/css/styles.css]: Please remove commented code

**Comment by megame24** [comment]
Don't you feel that the delete modal should have more padding?

**Comment by Bobjayafam** [comment]
> Don't you feel that the delete modal should have more padding?

I have worked on this. The screenshot has also been updated

**Comment by megame24** [comment]
> > Don't you feel that the delete modal should have more padding?
> 
> I have worked on this. The screenshot has also been updated

If you've implemented this feedback, it has not reflected in the screenshot. That cancel and delete buttons are still positioned at the edges of the modal

**Comment by Bobjayafam** [comment]
> > > Don't you feel that the delete modal should have more padding?
> > 
> > 
> > I have worked on this. The screenshot has also been updated
> 
> If you've implemented this feedback, it has not reflected in the screenshot. That cancel and delete buttons are still positioned at the edges of the modal

Sorry about that, I didn't update the screenshot. It's OK now.

**Review by megame24 — APPROVED** [review]
Looks good to merge, good job.

**Pull request merged**